packages/ui/Icon/Icon.js

```diff
@@ -141,11 +150,17 @@ const Icon = kind({
 	},
 
 	computed: {
-		className: ({children: icon, iconList, pressed, size, styler}) => styler.append({
-			// If the icon isn't in our known set, apply our custom font class
-			dingbat: !(icon in iconList),
-			pressed
-		}, size),
+		className: ({children: icon, flip, iconList, pressed, size, styler}) => {
+			return styler.append(
+				{
+					// If the icon isn't in our known set, apply our custom font class
+					dingbat: !(icon in iconList),
+					pressed
+				},
+				flip ? `flip${cap(flip)}` : null,
```

Comment:
Curious why we don't just use `flip-${flip}` and update the class names.  I guess it's really not a big deal!

Reply:
Yeah, that one probably would have been good to catch. More in-line with our other class names.

Reply:
Nevermind! I got confused for a minute. We use camelcase CSS classes everywhere. This allows us to refer to them conveniently in JavaScript: e.g. `css.flipHorizontal` vs `css['flip-horizontal']` (since `css.flip-horizontal` isn't permissible).

LESS Mixins, on the other hand, were grandfathered into the hyphen naming convention, which works out because you _can't_ refer to those from within our JavaScript.